### ISSUE TITLE
508-defect-2: Forms should prioritize choices or inputs that reduce the total amount of burden on the user

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#43256]
## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Josh

## User Story or Problem Statement

As a user with a screen reader, motor disabilities, low vision, or cognitive disabilities, I want it to be more explicitly clear that I can skip parts of a form.

## Details

This is more of a research request than an actual bug. Specifically, there's a note on step 3 of 6 regarding the deceased Veteran's military history that says users can skip filling in service periods with a DD214 form instead. For users with disabilities, being able to skip large chunks of forms can mean the difference of several hours (if not days) of labor. 

That being said, a couple of issues *may* exist with the current design:
- Visually, it's a low level alert given it uses the language **note** with a light orange background; for colorblind and screen reader users, the level of importance will be limited to "note" (assuming the original visual intent was to have orange, as opposed to say light blue, to indicate a higher level of importance than just a note)
- Screen reader users may completely skip over it if they're accustomed to using the tab key to quickly navigate through a form. They may end up having to labor over many extra fields as a result.

## Acceptance Criteria

- [ ] Team reviews design intent of the note 
- [ ] Provide options for addressing feedback, including design approaches

## Steps to Recreate

1. Go to step 3 of 6: military history
2. Turn on your screen reader of choice
3. Confirm that the note does not announce to screen reader users

## Proposed Solution (if known)

If it's determined that this is an important UX decision to reconsider; we may want to explore providing an explicit question up front asking if the user has a DD214, and if they reply yes, fast tracking them past the service fields (which may be more difficult for a family member or someone other than the Veteran to recall).

## Screenshots or Trace Logs

https://user-images.githubusercontent.com/14154792/175334942-ec935293-c718-47e0-9a1c-f86797d26174.mov

<img width="1428" alt="Screen Shot 2022-06-23 at 11 27 58 AM" src="https://user-images.githubusercontent.com/14154792/175337126-cec18533-c327-409a-80fd-d605b975d214.png">

<img width="602" alt="Screen Shot 2022-06-24 at 12 43 59 PM" src="https://user-images.githubusercontent.com/14154792/175604281-df5a1ce4-d1df-4bb9-9a6e-18e45a0f0f17.png">

**Comment by joshkimux** [comment]
@skylerschain could use your design 👀 here when you have the bandwidth

**Comment by joshkimux** [comment]
@jacobworrell I can explain this! If we make a design change here, the launchblocking issue related to service periods being opened and submitted blank may no longer be an issue. Specifically, if uploading a DD214 _earlier_ in the process can remove the need to input service history, our approach to how we design that page may change.

**Comment by jsokohl** [comment]
We might need to balance some levels of trying to achieve a zero defect approach can have. Keep in mind that designing explicitly to adjust that background color for achromatopsia means we are designing for an extremely small population of people 1 in 33,000 in the USA have congenital achromatopsia, in which case they actually see no grays. Only those with trauma-induced achromatopsia would see the page as the screenshot shows, according to [.](http://www.achromat.info/what_is_achromatopsia).

**Comment by joshkimux** [comment]
@jsokohl , just for clarification, the issue is only partially related to the color (which also applies to folks who use greyscale filters on their devices). 

To be more direct, I was hoping to get your thoughts on if an alternative design pattern (e.g. providing an opportunity to upload the DD214 earlier in the form) could save the user time from having to fill out the service record fields, especially if they're a caregiver or family member.

**Comment by joshkimux** [comment]
@skylerschain feel free to prioritize this when appropriate 🙏 
this one will likely need some more context digging to get an answer to, specifically:
- is the assumption that uploading a DD214 (or other proof of service) is easier and results in more accurate submissions for Veterans/caregivers/family members than manually inputting service dates?
- if so, is it technically feasible to ask the user if they want to upload a DD214 or manually input information ahead of time to branch the choices so users who upload a DD214 don't have to see/fill out their service record manually?

**Comment by jacobworrell** [comment]
@skylerschain and @jerekshoe we can consider the two questions asked from Josh in his most recent comment our questions for the spike.

**Comment by skylerschain** [comment]
[Design spec](https://www.sketch.com/s/4c0aad2f-9660-4944-9fcd-9de1d122ab8b/p/4DC554CF-886A-473E-B2E6-3C7D765E5BE5/canvas) created for this.

**Comment by fiorella-io** [comment]
This defect will be fixed in the redesigned Burial form. It'll follow Skyler's proposed design but with the upload happening on step 5.

Design:
- [New DD214 question](https://www.sketch.com/s/de782a35-e147-4c32-a2a8-ba53071ec8e7/v/1RQjx5/a/4oEjymo)
- [New DD214 document upload section on step 5](https://www.sketch.com/s/de782a35-e147-4c32-a2a8-ba53071ec8e7/a/OeW2Mwq)

We might change the upload right after the question depending on the amount of effort it would take
